scripts/busybox-ping-test/README.md

```diff
@@ -11,24 +11,27 @@
 - [Running Test](#Running-Test)
 
 ## Introduction
-This is an end to end test where we test two Busybox containers, hosted on same or different hosts, ping each other.The connectivity between the two Busybox containers
-is provided by Alcor services and Alcor Control Agent.
+This is an end to end test where we test two Busybox containers, hosted on same or different hosts, ping each other.The connectivity between the two Busybox containers is provided by Alcor services and Alcor Control Agent.
 
-## Key Features
-  - Builds Alcor and docker images for all alcor services (optional)
+## Key Features of Test Script
+  - Builds Alcor and docker images for all alcor services (optional).
   - Stops and removes existing alcor services and starts the alcor services afresh.
-  - Checks if the  Alcor Control Agents are running on the hosts over which Busybox containers are to be deployed.
+  - Stops any currently running ACA on target computers where Busybox containers are to be deployed.
+  - Clears any existing busybox containers on the target hosts.
+  - Checks if the Alcor Control Agents are running on the target hosts.
```

Comment:
What should happen if they are not running? Would the pipeline start them if they are mot already running?